src/questions/javascript-questions.md

```diff
@@ -64,3 +64,5 @@ console.log("hello" || "world")
 console.log("foo" && "bar")
 ```
 * Write an immediately invoked function expression (IIFE)
+* What is the difference between a deep copy and a shallow copy of a JavaScript object?
```

Comment:
I personally against the questions about `knowing terms`, except the case it's part of the language

what you're going to do if on an interview I would say `I've never heard such words` ?

Reply:
I often interview people who don't know certain terms. I continue the conversation either skipping over to the next question or I define what I'm asking about and then move onto the second part. Sometimes, people know the topic but just don't know the technical term.